src/TrueLayer/PaymentsProviders/Model/AuthorizationFlow.cs

```diff
@@ -0,0 +1,15 @@
+using System.Collections.Generic;
+using TrueLayer.Models;
+
+namespace TrueLayer.PaymentsProviders.Model;
+
+public record AuthorizationFlow(AuthorizationFlowConfiguration Configuration);
```

Comment:
Happy to reuse existing models, I've seen similar ones, but slightly different from what we use in payments-specific auth flows.

Reply:
We should probably create a base AuthFlow model that every other model derives from in case they differ.
If not in the common models yet, it is fine to have its own one for now, just to be consistent.